plugin-localization.md

```diff
@@ -33,6 +35,19 @@ The **lang.php** file should define and return an array of any depth, for exampl
             'tagline' => 'Getting back to basics'
         ]
     ];
+    
+The **validation.php** file has a similar structure to the **lang.php**  and is used to specify your [custom validation](https://octobercms.com/docs/services/validation#localization) messages in a language file, for example:
+
+    <?php
+
+    return [
+    'custom' => [
```

Comment:
I'm talking about the changes made in #160. Also, just noticed there were code formatting problems in the example from that PR as well - fixed in https://github.com/octobercms/docs/commit/2bed49de3abc7caafc18260c7f26114bedaa1aa5. The changes it needs to reflect are the removal of the `custom` element to make the validation work as stated in #160, and the code formatting issue is the improper indentation.